ci-operator/config/openshift/external-dns/openshift-external-dns-release-1.1.yaml

```diff
@@ -0,0 +1,38 @@
+base_images:
+  base:
+    name: "4.12"
+    namespace: ocp
+    tag: base
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: Dockerfile.openshift
+  from: base
+  to: external-dns
+promotion:
+  namespace: external-dns-operator
+  tag: "1.1"
```

Comment:
I think this promotes it into the OpenShift release.  The `promotion` tag is removed in https://github.com/openshift/release/commit/4c06fb054ae2d87564d65c68d0b32aee0a5c16ff